client/extensions/woocommerce/app/settings/taxes/taxes-rates.js

```diff
@@ -41,25 +41,21 @@ import QuerySettingsGeneral from 'woocommerce/components/query-settings-general'
 class TaxesRates extends Component {
 	static propTypes = {
 		onEnabledChange: PropTypes.func.isRequired,
-		site: PropTypes.shape( {
-			slug: PropTypes.string,
-		} ),
+		siteId: PropTypes.number.isRequired,
 	};
 
 	componentDidMount = () => {
-		const { address, loadedSettingsGeneral, loadedTaxRates, site } = this.props;
+		const { address, loadedSettingsGeneral, loadedTaxRates, siteId } = this.props;
 
-		if ( site && site.ID ) {
-			if ( loadedSettingsGeneral && ! loadedTaxRates ) {
-				this.props.fetchTaxRates( site.ID, address );
-			}
+		if ( loadedSettingsGeneral && ! loadedTaxRates ) {
+			this.props.fetchTaxRates( siteId, address );
 		}
 	};
 
 	componentWillReceiveProps = nextProps => {
 		if ( nextProps.loadedSettingsGeneral ) {
 			if ( ! nextProps.loadedTaxRates ) {
-				this.props.fetchTaxRates( nextProps.site.ID, nextProps.address );
+				this.props.fetchTaxRates( nextProps.siteId, nextProps.address );
```

Comment:
could these two `if` conditions be combined here? `if ( nextProps.loadedSettingsGeneral && ! nextProps.loadedTaxRates )`

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/19535/commits/4dc13074b82efbf815f60cf9475d4ecdb8eaf096